source/Cosmos.Core/Ports.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Cosmos.Core
+{
+    public static class Ports
+    {
+        /// <summary>
+        /// Reads a byte
+        /// </summary>
+        /// <param name="port"></param>
+        /// <returns></returns>
+        public static byte InB(ushort port)
```

Comment:
I prefer we go with the same approach we have for other drivers as well, which is that we define all the the IOPorts we need and then access the one we actually need. For example see https://github.com/CosmosOS/Cosmos/blob/a3d0a074f1f9db340694769116892aea925b8fde/source/Cosmos.Core/IOGroup/ATA.cs#L13 
That approach should be quicker since we arnt constantly creating new objects and less error prone than calculating the port address every time.